packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -20,6 +25,35 @@ export const OdieSendMessageButton = () => {
 	const isChatBusy = chat.status === 'loading' || chat.status === 'sending';
 	const [ isMessageSizeValid, setIsMessageSizeValid ] = useState( true );
 	const [ submitDisabled, setSubmitDisabled ] = useState( true );
+	const [ fileToUpload, setFileToUpload ] = useState< File | undefined >();
+	const onFilesDrop = ( files: File[] ) => {
+		const file = files?.[ 0 ];
+		if ( file && file.type.startsWith( 'image/' ) ) {
+			setFileToUpload( file );
+		}
+	};
```

Comment:
We should consume `useAttachFileToConversation` directly here. Passing around all these props is much dirtier imo.